test/util.test.js

```diff
@@ -520,6 +520,15 @@ describe('mergeOptions', function () {
     it('identifies a date string', function () {
       assert.equal(util.isDate(''), false);
     });
+
+    it('identifies non-dates', function () {
```

Comment:
😆 I just love how you copied over my 'usual suspects'. You'll encounter them as well in the `Network` unit tests.